curriculum/challenges/english/08-coding-interview-prep/rosetta-code/last-friday-of-each-month.md

```diff
@@ -0,0 +1,76 @@
+---
+id: 5a23c84252665b21eecc7edc
+title: Last Friday of each month
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+Write a function that returns the date of the last Friday of a given month for a given year.
+</section>
+
+## Instructions
+<section id='instructions'>
+
+</section>
+
+## Tests
+<section id='tests'>
+
+``` yml
+tests:
+  - text: <code>lastFriday</code> should be a function.
+    testString: assert(typeof lastFriday == 'function', '<code>lastFriday</code> should be a function.');
+  - text: <code>lastFriday(2018, 1)</code> should return a number.
```

Comment:
I think the next challenge takes care that a number is returned. What do you think of this and the other comments? @scissorsneedfoodtoo